src/components/Tables/MeetingVotesTable/MeetingVotesTable.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { MeetingVotesTableRow } from "../MeetingVotesTableRow";
+import { EmptyRow } from "../EmptyRow";
+import { MeetingVote } from "../../Shared/Types/MeetingVote";
+import "@mozilla-protocol/core/protocol/css/protocol.css";
+
+type MeetingVotesTableProps = {
+  /** the matter being voted upon */
+  votesPage: MeetingVote[];
+};
+
+function renderEmpty(index: number) {
+  return <EmptyRow key={`empty-meeting-table-row-${index}`} index={index} />;
+}
+
+function renderRows(votesPage: MeetingVote[]) {
+  const isEmpty = !votesPage || !votesPage.length || votesPage.length === 0;
+  return (
+    <tbody>
+      {isEmpty && renderEmpty(0)}
+      {votesPage.map((meetingVotes: any, index: number) => {
+        if (!meetingVotes.matter || !meetingVotes.votes) return renderEmpty(index);
+        const legislationName = meetingVotes.matter.name;
+        const councilDecision = meetingVotes.council_decision;
+        const legislationLink = `/matters/${meetingVotes.matter.id}`;
+        const meetingDate = meetingVotes.date;
+
+        return (
+          <MeetingVotesTableRow
+            key={`meeting-voting-table-row-${index}`}
+            index={index}
+            legislationName={legislationName}
+            councilDecision={councilDecision}
+            legislationLink={legislationLink}
+            meetingDate={meetingDate}
+            votes={meetingVotes.votes}
+          />
+        );
+      })}
+    </tbody>
+  );
+}
+
+function renderColGroup() {
+  return (
+    <colgroup>
+      <col style={{ width: "30%" }} />
+      <col style={{ width: "15%" }} />
+      <col style={{ width: "20%" }} />
+      <col style={{ width: "35%" }} />
+    </colgroup>
+  );
+}
+
+function renderHeaders() {
+  return (
+    <thead>
+      <tr>
+        <th scope="col">Matter</th>
+        <th scope="col">Date</th>
+        <th scope="col">Decision</th>
+        <th scope="col">Votes</th>
+      </tr>
+    </thead>
+  );
+}
+
+const VotingTable = ({ votesPage }: MeetingVotesTableProps) => {
+  return (
+    <table className="mzp-u-data-table" style={{ width: "100%" }}>
+      <caption>Click each row to see how council members voted</caption>
```

Comment:
I wonder if instead of text above the table tellings users to click on each row to see more information it would be a better UX experience to have a chevron icon at the beginning of the row that would indicate to the user that there is more information in each row without having to explicitly say so?

@tohuynh @JacksonMaxfield @BrianL3 Thoughts?

Something like this btw:

![image](https://user-images.githubusercontent.com/39639992/116799200-f31fa900-aaab-11eb-837f-a1be2b137ee1.png)
![image](https://user-images.githubusercontent.com/39639992/116799197-e9964100-aaab-11eb-93d0-4fb3763bbe37.png)

Reply:
I say both! Have the text at the top and have the expansion icons / behavior

Reply:
We discussed have a chevron, but it starts to look bad if you have one at the start of each  and every row.  It's the sort of pattern that, once revealed, the user does not need to be reminded of.  I was actually even thinking about hiding the caption text inside a little ? icon and have it revealed on click.